pkg/util/glog/glog.go

```diff
@@ -0,0 +1,97 @@
+package glog
```

Comment:
It will not grow.  I'll a comment to that effect - this is a transitional
tool for people who need to bulk convert glog to user friendly.  The real
solution is to pass an output stream or other interface.

On Sat, May 7, 2016 at 7:51 PM, Ben Parees <notifications@github.com> wrote:

> In pkg/util/glog/glog.go
> <https://github.com/openshift/origin/pull/8786#discussion_r62424358>:
>
> > @@ -0,0 +1,97 @@
> > +package glog
>
> nit picky but if this is going to grow into a generic log wrapper, i don't
> think we should be calling it glog. it's going to make for confusing code
> (is this code using the wrapper or the raw glog? have to go check the
> import to know)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8786/files/9725d5171cdd040a8f619a1a7d7b332d9228209b#r62424358>
>